packages/infra/infra-shared/src/main.ts

```diff
@@ -11,11 +11,14 @@ import { EnvMainStack } from './stacks/main';
 import { EnvDbStack } from './stacks/db';
 import { EnvComponentsStack } from './stacks/components';
 import { EnvCiStack } from './stacks/ci';
+import { BootstrapStack } from './stacks/bootstrap';
 
 (async () => {
   const envSettings = await loadEnvSettings();
   const app = new App();
 
+  new BootstrapStack(app, 'sb-bootstrap');
```

Comment:
Maybe let's keep the convention as `new BootstrapStack(app, getGlobalStackName('BoostrapStack', envSettings));`

Reply:
We can't really. Think of this stack similarily to the `cdk-bootstrap` stack. There needs to be exactly one instance of it for the whole AWS account. Even if you have multiple projects. That's why I hardcoded the name.